extensions/storages/schemas/s3.cos.yaml

```diff
@@ -0,0 +1,13 @@
+type: object
+properties:
+  provider:
+    const: s3
+  bucket:
+    type: string
```

Comment:
This should have way more validations: https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html#general-purpose-bucket-names

Reply:
I believe this is not worth the effort.

In the worst-case scenario, a developer may experience a runtime exception from the AWS SDK.